UnitTesting/ProcedureUnitTests.cs

```diff
@@ -143,5 +143,237 @@ public SegmentEventsEntry GetSegmentEventsObject(string eve,double inn, double o
             obj.TimeStamp = Convert.ToDateTime(time);
             return obj;
         }
+
+
+        [Test]
+        public void TestsummaryUsageDaily()
+        {
+            var build = new ConfigurationBuilder();
+            IConfiguration _config = build.Build();
+            var expected = GetTestExpectedSummaryDaily();
+            var dataset = GetTestDataSummaryDaily();
+            Procedures proc = new Procedures(new DatabaseContext(), _config);
```

Comment:
Just a suggestion (wont stop you from merging) 
you can actually use *var* for 
`var _config = build.Build();` and
` var proc = new Procedures(new DatabaseContext(), _config);`